spec/function.spec.js

```diff
@@ -0,0 +1,127 @@
+const {
+  resetConstants,
+  stringToArray,
+  multiplyCheck,
+  findLetterScore,
+  scoreMultiplicatorizer,
+  tallyScore
+} = require('../src/scrabble')
+
+describe('stringToArray', () => {
+  beforeEach(() => {
+    resetConstants()
```

Comment:
I'm getting PTSD from this because it caused so many weird errors for me when running the tests until I figured out that it had to do with that – wish I would've seen this earlier / or your question thread because it was a pain

Reply:
Definitely! I consistently had half the tests in extension fail which led me to create `resetConstants()` it wasn't till later I also added it to use in the spec files after some help from Jules and Lewis.